test/array-structures.test.js

```diff
@@ -0,0 +1,425 @@
+const { deepStrictEqual, strictEqual } = require('assert')
+const { valueByImplType, isStatusCodeError } = require('./testutils')
+
+exports.test = function (setup, implType) {
```

Comment:
Should this not be module.exports.test or is this mocha specific?

Reply:
:man_facepalming: oookkkk :-D